src/nodes/display/GTAONode.js

```diff
@@ -53,12 +54,6 @@ class GTAONode extends TempNode {
 
 	}
 
-	getTextureNode() {
```

Comment:
I made some changes to [this PR](https://github.com/mrdoob/three.js/pull/28863), I think it's important to keep `getTextureNode()` for external manipulation level, like adding a custom Denoise filter or just a blur without having to do a new render-pass.

I'm removing `blendIntensity` too, now with TSL I think it's better that this level of abstraction stays outside the internal Nodes, intensity, blend are things that would only bring more load to the examples since we could use `blendIntensity.mix( beauty, beauty.mul( ao )` externally if necessary.

Reply:
Yeah, you think the same think <3  https://github.com/mrdoob/three.js/pull/28851#issuecomment-2223704038
I'm reading the comments now :)

Reply:
I'm fine with your suggested changes. I've only added `blendIntensity` now so the implementation can be considered as "complete" and usable in apps.